app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MediaRanker</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+  <header class="app-header">
+    <h1><a href="/">Media Ranker</a>
+    <small>Ranking the Best of Everything</small></h1>
+
+    <nav class="app-header-nav">
+      <div class="link-header"><%= link_to 'View top media', works_top_path %></div>
```

Comment:
I'm being nitpicky, but this link would be better going to `root_path` instead. When we look at the URL after clicking on this link, it has `/works/top` attached to it... which is fine, but root is better and cleaner!